src/crypto-api/CrossSigningInfo.ts

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2023 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { CrossSigningKey } from "../crypto-api";
+
+/**
+ * Information about a user's cross-signing keys
+ */
+export interface CrossSigningInfo {
+    /**
+     * Return the public key of the cross singing key
+     *
+     * @param crossSigningKey - the type of cross signing key wanted
+     *
+     * @returns
+     */
+    getId(crossSigningKey?: CrossSigningKey): string | null;
```

Comment:
Since we're creating a new interface anyway, maybe we can make a better name for it? `getPublicKey` maybe?